Userland/Libraries/LibWeb/CSS/SelectorEngine.cpp

```diff
@@ -44,9 +44,9 @@ static inline bool matches_lang_pseudo_class(DOM::Element const& element, Vector
             return false;
         if (language == "*"sv)
             return true;
-        if (!element_language.view().contains('-'))
-            return element_language.equals_ignoring_case(language);
-        auto parts = element_language.view().split_view('-');
+        if (!element_language.bytes_as_string_view().contains('-'))
+            return element_language.bytes_as_string_view().equals_ignoring_case(language);
+        auto parts = element_language.bytes_as_string_view().split_view('-');
```

Comment:
Feels like these should go through String's APIs rather than StringView's. e.g. the split here hides failures, whereas String's does not. Could even add a `String::for_each_split` that uses IterationDecision to break after the first split, since that's all that's wanted here.

Reply:
I've gone with `split_limit('-', 2)` for now. A per-split iterator would be nice though. :thinking: